.eslintrc.js

```diff
@@ -1,13 +1,12 @@
-const os = require('os');
-
 module.exports = {
   env: {
     es2021: true,
     node: true,
     jest: true,
   },
   ignorePatterns: ['**/functions/*.js'],
-  extends: ['next', 'airbnb', 'plugin:cypress/recommended'],
+  // Prettier plugin should be the last to always override preceding plugins
+  extends: ['next', 'airbnb', 'plugin:cypress/recommended', 'prettier'],
```

Comment:
are we sure we want prettier to be last? that means that if there's a rule that only exists within prettier then the entire project could be changed?